exp/templates/exp/_navigation.html

```diff
@@ -57,3 +55,8 @@
     </div><!-- /.navbar-collapse -->
   </div><!-- /.container-->
 </nav>
+<script type="application/javascript">
+    $(function () {
+        $('[data-toggle="tooltip"]').tooltip()
+    });
+</script>
```

Comment:
are we using tooltips on this page / does this end up covering other pages magically?

Reply:
It'll cover pages where this partial is injected - and it's just to create the bootstrap-skinned tooltips. This was just to provide a nice bootstrappy tooltip over the original 2FA badge, which we dropped, so I'll go ahead and get rid of this.